build.sh

```diff
@@ -2,6 +2,6 @@
 # Proof-of-concept mill build that can be used with ammonite.
 # TODO: make this single file and add maven version
 
-mill diagrammer.assembly
-cp out/diagrammer/assembly/dest/out.jar diagrammer.jar
+mill "diagrammer[2.12.7].assembly"
```

Comment:
This should still work out of the box in latest mill, no?

Maybe add a comment in build.sh w/the mill version it works with.

Reply:
Yes. in lasted mill(0.5.1, this works out of box.

Reply:
As in can you leave the line as `mill diagrammer.assembly` and have it still work? It seems strange to have to hardcode the Scala version

Reply:
Done, but the method to find `out.jar`(avoid `2.12.7` in script) is a little ugly :p